tools/README.md

```diff
@@ -0,0 +1,73 @@
+# Readme for tools
+
+
+## Introducing grappler and tf2ngraph
+
+### Normal mode of operations
+The normal mode of operation of `ngrph_bridge` is:
+```
+import ngraph_bridge
+
+in0, in1, out0 = construct_network()
+sess = tf.Session()
+sess.run(out0, feed_dict = {in0:[2,3], in1:[4,5]})
+```
+
+The "normal" mode latches onto `Tensorflow` operations by registering `GraphOptimizationPass`s such as `NGraphRewritePass` and `NGraphVariableCapturePass`. It also register a custom op `NGraphEncapsulate`. Note that the graph rewriting and the actual execution all happens when `session.run` is called
```

Comment:
`GraphOptimizationPass`s ----extra s

Reply:
Ohh great grammar catch. Possessive's for words ending with double `ss` doesn't need an `s`